pkg/util/config/config.go

```diff
@@ -25,6 +25,7 @@ type Resources struct {
 	ClusterLabels           map[string]string `name:"cluster_labels" default:"application:spilo"`
 	ClusterNameLabel        string            `name:"cluster_name_label" default:"cluster-name"`
 	PodRoleLabel            string            `name:"pod_role_label" default:"spilo-role"`
+	PodToleration          	map[string]string `name:"toleration" default:""`
```

Comment:
I think this should correspond to the default value in configmap.yaml (i.e. "key:postgres,operator:Exists,effect:NoSchedule").

Reply:
After removing it from the general ConfigMap I think the default should be empty too.